StaticClasses/1/FindAndReplaceManager.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace StaticClasses
+{
+    internal static class FindAndReplaceManager
+    {
+        public static void FindNext(this string str, Book book)
```

Comment:
That's not right. Your method should be public static void FindNext(this string str).
That means you need to have a method in Book class which uses this static class method.